src/ace.F90

```diff
@@ -482,6 +482,15 @@ subroutine read_esz(nuc, data_0K)
       ! Continue reading elastic scattering and heating
       nuc % elastic = get_real(NE)
 
+      ! Determine if minimum/maximum energy for this nuclide is greater/less
+      ! than the previous
+      energy_min_neutron = max(energy_min_neutron, nuc%energy(1))
+      energy_max_neutron = min(energy_max_neutron, nuc%energy(NE))
+      if (nuc%energy(NE) < 20.0_8) then
```

Comment:
I suggest removing this 20 MeV hardcode, too, and print a message with the min and max energies (and the corresponding nuclides) that transport will be performed for if not all nuclides' bounds agree.  Since we can now handle fixed sources > 20 Mev if the data is there, it would be frustrating to specify a 100 MeV source in a problem with some nuclides that don't have data that high.  You get error when checking the source energy w/o getting the offending nuclide